src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -15,11 +15,22 @@
   (let [parsed-keypairs (data-store/rpc->keypairs keypairs)
         default-key-uid (->> parsed-keypairs
                              (some #(when (= (:type %) :profile) %))
-                             :key-uid)]
+                             :key-uid)
+        keycard?        (get-in db [:profile/profile :keycard-pairing])
+        keypair         (first parsed-keypairs)]
     {:db (-> db
              (assoc-in [:wallet :keypairs] (utils.collection/index-by :key-uid parsed-keypairs))
-             (assoc-in [:wallet :ui :create-account :selected-keypair-uid] default-key-uid))}))
-
+             (assoc-in [:wallet :ui :create-account :selected-keypair-uid] default-key-uid))
+     ;; this is a temporary solution, should be done properly in statu-go instead
```

Comment:
i can remove this comment then, so that it doesn't embarrass anyone :) better solution would be to implement this in status-go during account creation, no need to have an extra call from "client", probably this will be implemented by desktop during onboarding refactoring

Reply:
Your explanation is already clearer than the comment in the code because I see now what you mean by doing this at account creation. Thanks, up to you to judge what's best to do @flexsurfer